Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "mcp_server_brave_search"
-version = "0.0.2"
+version = "0.0.3"
```

```suggestion
version = "0.0.2"
```

We bump versions separately, I'll make sure a new version is released once this lands